### PR TITLE
Add Soketi Service

### PR DESCRIPTION
**Pull request by Nathanjms**
## Changes
- Added service templates and logo for Soketi
    - I couldn't find an SVG so have used a PNG, I hope this is okay!

I have this working as a docker compose by using `https://ws.mydomain.com:6001` as the domain.

### CODE REVIEW

File: templates/compose/soketi.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    image: 'quay.io/soketi/soketi:1.6-16-alpine'
+    expose:
+      - 6001
+    environment:
+      - 'SOKETI_DEBUG=${DEBUG:-0}'
+      - 'SOKETI_DEFAULT_APP_ID=${SOKETI_APP_ID}'
+      - 'SOKETI_DEFAULT_APP_KEY=${SOKETI_APP_KEY}'
+      - 'SOKETI_DEFAULT_APP_SECRET=${SOKETI_APP_SECRET}'
+      - 'SOKETI_PUSHER_SCHEME=${SOKETI_PUSHER_SCHEME:-https}'
+      - 'SOKETI_DEFAULT_APP_ENABLE_CLIENT_MESSAGES=${DEFAULT_APP_ENABLE_CLIENT_MESSAGES}'
```
Comment (rennokki) [inline templates/compose/soketi.yaml:20]: Add a new line at the very bottom.

File: templates/compose/soketi.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# tags: websockets,open,source,messaging
+# logo: svgs/soketi.png
+# port: 6001
+
+services:
+  soketi:
+    container_name: soketi_server
+    restart: unless-stopped
+    image: 'quay.io/soketi/soketi:1.6-16-alpine'
+    expose:
```
Comment (rennokki) [inline templates/compose/soketi.yaml:12]: No need for expose 6001.

File: templates/compose/soketi.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# documentation: https://docs.soketi.app
+# slogan: Soketi is your simple, fast, and resilient open-source WebSockets server.
+# tags: websockets,open,source,messaging
+# logo: svgs/soketi.png
+# port: 6001
+
+services:
+  soketi:
+    container_name: soketi_server
```
Comment (rennokki) [inline templates/compose/soketi.yaml:9]: No need for container name

File: templates/compose/soketi.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# documentation: https://docs.soketi.app
+# slogan: Soketi is your simple, fast, and resilient open-source WebSockets server.
+# tags: websockets,open,source,messaging
+# logo: svgs/soketi.png
```
Comment (rennokki) [inline templates/compose/soketi.yaml:4]: I recommend the new logo: https://avatars.githubusercontent.com/u/70626942?s=400&u=4b09731b5a6771c46cba09662109acd154cd2723&v=4

**Comment by peaklabs-dev** [comment]
Thanks for the PR. I will fix the things mentioned @rennokki thanks for the review btw.

**Pull request merged**